app/views/EmployeeFirstFurloughedView.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import pages.EmployeeFirstFurloughedPage
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    errorSummary: components.errorSummary,
+    inputDate: components.inputDate,
+    formHelper: FormWithCSRF,
+    button: components.button,
+    p: components.p,
+    h1: components.h1
+)
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages)
+
+@govukLayout(pageTitle = Some(title(form, "employeeFirstFurloughed.title")), timeout = false, thisPage = EmployeeFirstFurloughedPage ) {
+
+    @formHelper(action = EmployeeFirstFurloughedController.onSubmit(), 'autoComplete -> "off") {
+
+      @h1(messages("employeeFirstFurloughed.heading"))
+
+       @p(Html(messages("employeeFirstFurloughed.p1")))
+
+       @p(Html(messages("employeeFirstFurloughed.p2")))
+
+        @errorSummary(form.errors, Some("date-value.day"))
```

Comment:
I think the error summary should go above the h1 unless the calculator works differently